react/features/filmstrip/components/Filmstrip.native.js

```diff
@@ -86,30 +89,30 @@ class Filmstrip extends Component<*> {
      *
      * @param {Participant[]} participants - The array of {@code Participant}s
      * to sort in display order.
+     * @param {boolean} isNarrowAspectRatio_ - Indicates if the aspect ratio is
```

Comment:
What's the deal with the underscore after or before a var name ?

Reply:
It feels like another unspoken coding style convention to me

Reply:
It also surprised me. I copied it from elsewhere in the same file. The _ is used to avoid overriding the imported function.